client/state/themes/test/actions.js

```diff
@@ -229,19 +229,19 @@ describe( 'actions', () => {
 	} );
 
 	describe( '#requestTheme()', () => {
-		useNock( ( nock ) => {
-			nock( 'https://public-api.wordpress.com:443' )
-				.persist()
-				.get( '/rest/v1.2/themes/twentysixteen' )
-				.reply( 200, { id: 'twentysixteen', title: 'Twenty Sixteen' } )
-				.get( '/rest/v1.2/themes/twentyumpteen' )
-				.reply( 404, {
-					error: 'unknown_theme',
-					message: 'Unknown theme'
-				} );
-		} );
-
 		context( 'with a wpcom site', () => {
+			useNock( ( nock ) => {
+				nock( 'https://public-api.wordpress.com:443' )
+					.persist()
```

Comment:
I don't think these `persist` calls are generally necessary unless you want the same result multiple times.